### PR TITLE
Trello-658 Add Variables for Logging Docker Events and API Calls

### PR DESCRIPTION
**Pull request by bmcelvee**
https://trello.com/c/kjrZKSly/658-document-sccfsi-docker-events-and-api-calls-need-to-be-loggedloggingepic-ois-agl-exp

https://trello.com/c/qxoFsKz9/497-8-sccfsi-docker-events-and-api-calls-need-to-be-loggedloggingepic-ois-agl-exp

### CODE REVIEW

**Comment by bmcelvee** [comment]
@josefkarasek PTAL

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]
|Added `openshift_logging_fluentd_audit_container_engine`, `openshift_logging_fluentd_audit_file`, and `openshift_logging_fluentd_audit_pos_file` parameters to the
xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Specifying Logging Ansible Variables] section.
%

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review 🍁

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -288,6 +288,17 @@ starting up, using this may cause a delay in ES receiving current log records.
 |`openshift_logging_fluentd_hosts`
 |List of nodes that should be labeled for Fluentd to be deployed.
 
+|`openshift_logging_fluentd_audit_container_engine`
+|When `openshift_logging_fluentd_audit_container_engine` is set to `true`, the
+audit log of the container engine will be collected and stored in ES.
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: s/will be collected/is collected ??

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -288,6 +288,17 @@ starting up, using this may cause a delay in ES receiving current log records.
...
+
+|`openshift_logging_fluentd_audit_file`
+|Location of audit log file. The default is `/var/log/audit/audit.log`.
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: s/audit log file/the audit log file

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -288,6 +288,17 @@ starting up, using this may cause a delay in ES receiving current log records.
...
+
+|`openshift_logging_fluentd_audit_pos_file`
+|Location of Fluentd `in_tail` position file for the audit log file. The default is
+`/var/log/audit/audit.log.pos`
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: Period after `/var/log/audit/audit.log.pos`

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -288,6 +288,17 @@ starting up, using this may cause a delay in ES receiving current log records.
...
 
+|`openshift_logging_fluentd_audit_container_engine`
+|When `openshift_logging_fluentd_audit_container_engine` is set to `true`, the
+audit log of the container engine will be collected and stored in ES.
+
+|`openshift_logging_fluentd_audit_file`
+|Location of audit log file. The default is `/var/log/audit/audit.log`.
+
+|`openshift_logging_fluentd_audit_pos_file`
+|Location of Fluentd `in_tail` position file for the audit log file. The default is
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: Location of **the** Fluentd... ?

**Comment by mburke5678** [comment]
@bmcelvee Couple o'nits. LGTM

**Comment by bmcelvee** [comment]
Thanks, @mburke5678! 🍀

**Pull request merged**